Components/navigation/navbar.css

```diff
@@ -0,0 +1,59 @@
+.nav-bar {
+  width: 100%;
+  height: 5rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 1rem;
+  z-index: 1;
+}
+
+.boxShadow {
+  box-shadow: 0px 2px 10px 4px rgb(0 0 0 / 5%);
+}
+
+.nav-innerContainer {
```

Comment:
Your class names are very uneven, you have multiple types of naming in classes, please use just one.